DSE_5.1_Release_Notes.md

```diff
@@ -2,6 +2,28 @@
 DSE 5.1.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
 
+# Release notes for 5.1.28
+17 January 2022
+
+## Components versions for DSE 5.1.28
+ * Apache Solr™ 6.0.1.0.2882
+ * Apache Spark™ 2.0.2.42
+ * Apache TinkerPop™ 3.2.11-20210716-faea8d16
+ * Apache Tomcat® 8.5.72&ast;
+ * DSE Java Driver 1.8.3-dse+20201217
+ * Netty 4.0.54.1.dse
+ * Spark JobServer 0.6.2.240
+
+## 5.1.28 DSE Cassandra
+* Await timeout for shutting down non periodic tasks is now configurable with the new jvm option {{cassandra.non_periodic_tasks_shutdown_timeout_in_minutes}}. When timeout is reached, force shutdown those tasks. (DSP-22241)
```

Comment:
Here we should replace jira formatting by Markdown formatting (i.e. replace {{ and }} by ` )

Reply:
Ok. Will do.